### ISSUE TITLE
Install progress status bar color doesn't meet accessibility requirement

### ISSUE THREAD

**Issue by SteveL-MSFT** [wixtoolset/issues#6583]
## Bugs

If this issue is a bug:

* Which version of WiX are you building with?

> 3.1.1

* Which version of Visual Studio are you building with (if any)?

> not using Visual Studio

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> N/A

* Which version of .NET are you building with?

> .NET 6 rc1

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Microsoft Windows 10.0.19043

* Describe the problem and the steps to reproduce it.

> Install https://github.com/PowerShell/PowerShell/releases/download/v7.1.4/PowerShell-7.1.4-win-x64.msi

* Describe the behavior you expected and how it differed from the actual behavior.

> The green progress bar during install doesn't meeting the accessibility requirement of having 3:1 contrast

![image](https://user-images.githubusercontent.com/11859881/134076530-aa189a7a-5043-4d59-af8b-1d8e520172f5.png)

**Comment by barnson** [comment]
Windows Installer doesn't provide control over progress bar colors.
